service/src/main/java/org/lfenergy/compas/scl/data/service/impl/CompasSclDataServiceImpl.java

```diff
@@ -200,6 +213,10 @@ public void delete(SclFileType type, UUID id, Version version) {
         repository.delete(type, id, version);
     }
 
+    private boolean hasDuplicateSclName(SclFileType type, String name) {
```

Comment:
Why the need for a private method here? There is only 1 simple line in there.

Reply:
True, good one!